src/Analyzer.Cli/CommandLineParser.cs

```diff
@@ -350,42 +354,74 @@ private bool IsValidTemplate(string template)
             return false;
         }
 
-        private static IReportWriter GetReportWriter(ReportFormat reportFormat, FileInfo outputFile, string rootFolder = null)
-        {
-            switch (reportFormat)
-            {
-                case ReportFormat.Sarif:
-                    return new SarifReportWriter((FileInfoBase)outputFile, rootFolder);
-                case ReportFormat.Console:
-                    return new ConsoleReportWriter();
-                default:
-                    return new ConsoleReportWriter();
-            }
-        }
+        private static IReportWriter GetReportWriter(ReportFormat reportFormat, FileInfo outputFile, string rootFolder = null) =>
+            reportFormat switch {
```

Comment:
Why make this a switch and not an if/else? I think prior it was to make each case be explicitly defined or for the scenario where more cases would be added